python/tox.ini

```diff
@@ -9,7 +9,7 @@ envlist =
 [testenv]
 commands = 
     pytest -rsXx {posargs:tests}
-    pip wheel . -w {toxworkdir}/wheelhouse
+    python tests/package_wheels.py {toxworkdir}/wheelhouse
```

Comment:
Unrelated to this PR, but I'm not a huge fan of the wheel building in the same tox env as the tests.  I'd be happier if it was a different env.